### ISSUE TITLE
Injected css is overwritten Chrome63

### ISSUE THREAD

**Issue by tgfjt** [ZenHubIO/support#1272]
Hi.:)

Unfortunately,  `.container` is not full-width on Chrome63 like this below.
![image](https://user-images.githubusercontent.com/2628239/31698293-c6fc95d0-b3f7-11e7-860c-ff94decc255a.png)

Finally I found out that it seems to be loaded like default stylesheet and overwritten.

### Chrome62

![image](https://user-images.githubusercontent.com/2628239/31698174-009aef54-b3f7-11e7-9441-58eedbc3965d.png)

### Chrome63

![image](https://user-images.githubusercontent.com/2628239/31698275-97bb81f0-b3f7-11e7-90b3-162c9d3f43dd.png)

**Comment by devinmcinnis** [comment]
Hi @tgfjt, 

Thanks for the bug report. After looking into the issue, it looks to be a CSS rendering problem specific to Chrome Canary - a platform that we do not support. Two options that we have available are:

- the [ZenHub extension on Chrome](https://chrome.google.com/webstore/detail/zenhub-for-github/ogcgkffhplmphkaahpmffcafajaocjbd)
- the [ZenHub webapp](https://app.zenhub.com/)

Devin

**Comment by tgfjt** [comment]
@devinmcinnis Thank for your reply!
Do you mean that ZenHub do not support futuristic stable Chrome because it's canary?

**Comment by devinmcinnis** [comment]
Correct. ZenHub does not support Canary because it is not a stable release.
